.github/workflows/nightly_nosmp.yml

```diff
@@ -0,0 +1,119 @@
+name: Tests with SMP support (nightly)
```

Comment:
Either the filename is wrong or the name

Reply:
Oh, a typo... I fixed it.